cmd/juju/application/bundle.go

```diff
@@ -92,27 +87,47 @@ func deployBundle(
 	}
 	var verifyError error
 	if bundleDir == "" {
-		// Process includes in the bundle data.
-		if err := processBundleIncludes(ctx.Dir, data); err != nil {
-			return nil, errors.Annotate(err, "unable to process includes")
-		}
 		verifyError = data.Verify(verifyConstraints, verifyStorage, verifyDevices)
 	} else {
-		// Process includes in the bundle data.
-		if err := processBundleIncludes(bundleDir, data); err != nil {
-			return nil, errors.Annotate(err, "unable to process includes")
-		}
 		verifyError = data.VerifyLocal(bundleDir, verifyConstraints, verifyStorage, verifyDevices)
 	}
-	if verifyError != nil {
-		if verr, ok := verifyError.(*charm.VerificationError); ok {
-			errs := make([]string, len(verr.Errors))
-			for i, err := range verr.Errors {
-				errs[i] = err.Error()
-			}
-			return nil, errors.New("the provided bundle has the following errors:\n" + strings.Join(errs, "\n"))
+
+	if verr, ok := errors.Cause(verifyError).(*charm.VerificationError); ok {
+		errs := make([]string, len(verr.Errors))
+		for i, err := range verr.Errors {
+			errs[i] = err.Error()
 		}
-		return nil, errors.Trace(verifyError)
+		return errors.New("the provided bundle has the following errors:\n" + strings.Join(errs, "\n"))
+	}
+	if verifyError != nil {
```

Comment:
don't need this
errors.Trace(err) returns nil if err is nil

Reply:
Fixed!